### PR TITLE
Add support to export IPs for the UIPLog module

### PR DESCRIPTION
**Pull request by sepal**
IPs are personal information, therefore they should be exported when the user requests an export.
I'm not sure however, if there are multiple similar modules. I don't want to add support for all module, since this would make the code a bit complex.

### CODE REVIEW

File: gdpr_export.module [hunk 1]
```diff
@@ -231,6 +231,7 @@ function gdpr_export_form_user_profile_form_alter(&$form, &$form_state, $form_id
       '#type' => 'submit',
       '#value' => t('Export all data'),
       '#submit' => ['gdpr_export_user_data_submit'],
+      '#limit_validation_errors' => array(), // Ignore any required fields the user may or may not have selected.
```
Comment (klausi) [inline gdpr_export.module:234]: Comments should be on the line before the statement

File: gdpr_export.module [hunk 2]
```diff
@@ -239,5 +240,12 @@ function gdpr_export_form_user_profile_form_alter(&$form, &$form_state, $form_id
  * Submit callback for the gdpr export button on the user edit form.
  */
 function gdpr_export_user_data_submit(&$form, &$form_state) {
-  drupal_goto('user/' . $form['#user']->uid . '/gdpr_export');
+  // Remove destination if set
```
Comment (klausi) [inline gdpr_export.module:243]: Why? You are setting the destination anyway again in the redirect array, so you can leave it there?

File: gdpr_export.module (hunk 1 above)
Comment (sepal) [reply gdpr_export.module:234]: Oops, that change is from a different patch. Forgot to update this repo, see the issue here: https://www.drupal.org/project/gdpr_export/issues/2959890

File: gdpr_export.module (hunk 2 above)
Comment (sepal) [reply gdpr_export.module:243]: Again, form the same other issue: https://www.drupal.org/project/gdpr_export/issues/2959890

**Comment by sepal** [comment]
After reviewing the patch for the following issue, I'm wondering, if we also should move the UIPlog export into its own sub module. What do you think @klausi ?
https://www.drupal.org/project/gdpr_export/issues/2959945

**Comment by klausi** [comment]
I don't think another module is necessary. The code is not too long, so should be fine with the module_exists() call.

**Comment by sepal** [comment]
manually merged

**Pull request closed without merging**